packages/vue-output-target/vue-component-lib/utils.ts

```diff
@@ -59,8 +71,8 @@ export const defineContainer = <Props>(
     customElements.define(name, customElement);
   }
 
-  const Container = defineComponent<Props & InputProps>((props, { attrs, slots, emit }) => {
-    let modelPropValue = (props as any)[modelProp];
+  const Container = defineComponent<Props & InputProps>((props: any, { attrs, slots, emit }) => {
```

Comment:
Not blocking, but is there a way to infer/use some typing here, or would the type be way too large - I see it even includes onClick so my assumption is that it could be rather large and hard to add.

Reply:
Vue uses this `Props` type: https://github.com/vuejs/vue-next/blob/870f2a7ba35245fd8c008d2ff666ea130a7e4704/packages/runtime-core/src/apiDefineComponent.ts#L44

But `PropsOrPropOptions` is just an empty `{}`, though `ExtractPropTypes` looks like this: https://github.com/vuejs/vue-next/blob/870f2a7ba35245fd8c008d2ff666ea130a7e4704/packages/runtime-core/src/componentProps.ts#L106-L109

Reply:
Got ya, I checked it out and I see what you mean about `PropsOrPropOptions`. I'm okay with this. Thank you!